tests/data/gpkg_util_test.py

```diff
@@ -0,0 +1,84 @@
+import sqlite3
+import pandas as pd
+import numpy as np
+from geopandas import GeoDataFrame
+from shapely import Polygon
+from lukefi.metsi.data.formats import gpkg_util
+from tests.data import test_util
+
+
+class TestGeoPackageConversion(test_util.ConverterTestSuite):
+
+    GPKG_DB_PATH="tests\\data\\resources\\SMK_source.gpkg"
```

Comment:
I think this could be the culprit for loading the files with Fiona. Use forward slashes / for directory separator.